sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/.editorconfig

```diff
@@ -0,0 +1,199 @@
+# top-most EditorConfig file
```

Comment:
Please remove;  we'll want to conform to the standards from the repository root .editorconfig

Reply:
Makes sense. Removed it.